src/Feature/Editor/Editor.re

```diff
@@ -66,25 +68,27 @@ let getPrimaryCursor = model =>
 
 let getId = model => model.editorId;
 
-let pixelPositionToLineColumn =
-    (view, metrics: EditorMetrics.t, pixelX, pixelY) => {
-  let line = int_of_float((pixelY +. view.scrollY) /. metrics.lineHeight);
+let getLineHeight = editor => editor.font.measuredHeight;
```

Comment:
@lessp has been working on a line height setting I believe, and so removing the line height property entirely is pulling the rug out underneath that work a bit.

Reply:
This is fine, I haven't started any real work yet, more familiarising myself with the code base. 👍 Thanks for the consideration though! @glennsl

Reply:
Ah sorry @lessp (and thanks @glennsl for checking!)

This would actually be a good reference for where we use this - we use the `measuredHeight` as a proxy for line height, but it'd be to have a real configuration for this.

We'd want an additional `lineHeight` that we set per-editor (via a configuration), and we'd update our `getLineHeight` to account for it, ie:
```
let getLineHeight = editor => editor.font.measuredHeight *. editor.lineHeight;
```
(although we'd need to special case the `0` case - and just use `editor.font.measuredHeight` for that).